docs/markdown/snowplow_media_player_macro_docs.md

```diff
@@ -1,4 +1,130 @@
-{% docs macro_media_id_col %}
+{% docs macro_field %}
+{% raw %}
+This macro is used to define a path to a column either as a string or using a dictionary definition.
+
+On BigQuery, the `snowplow_utils.get_optional_fields` macro is used.
+
+#### Returns
+
+The query path for the field.
+
+#### Usage
+
+```sql
+select
+    ...,
+    {{ field('a.contexts_com_youtube_youtube_1[0]:playerId') }},
+    {{ field({ 'field': 'playerId', 'col_prefix': 'a.contexts_com_youtube_youtube_1' }) }},
+    from {{ var('snowplow__events') }} as a
+```
+
+{% endraw %}
+{% enddocs %}
+
+{% docs macro_media_ad_break_field %}
+{% raw %}
+This macro retrieves a property from the media ad break context entity.
+
+#### Returns
+
+The query path for the field.
+
+#### Usage
+
+```sql
+select
+    ...,
+    {{ media_ad_break_field('mb.name') }},
+    {{ media_ad_break_field({ 'field': 'name' }) }},
+    from {{ var('snowplow__events') }} as a
+```
+
+{% endraw %}
+{% enddocs %}
+
+{% docs macro_media_ad_field %}
+{% raw %}
+This macro retrieves a property from the media ad context entity.
+
+#### Returns
+
+The query path for the field.
+
+#### Usage
+
+```sql
+select
+    ...,
+    {{ media_ad_field('ma.name') }},
+    {{ media_ad_field({ 'field': 'name' }) }},
+    from {{ var('snowplow__events') }} as a
+```
+
+{% endraw %}
+{% enddocs %}
+
+{% docs macro_media_ad_quartile_event_field %}
+{% raw %}
+This macro retrieves a property from the media ad quartile self-describing event.
+
+#### Returns
+
+The query path for the field.
+
+#### Usage
+
+```sql
+select
+    ...,
+    {{ media_ad_quartile_event_field('mq.percent_progress') }},
+    {{ media_ad_quartile_event_field({ 'field': 'percent_progress' }) }},
+    from {{ var('snowplow__events') }} as a
+```
+
+{% endraw %}
+{% enddocs %}
+
+{% docs macro_media_event_type_field %}
+{% raw %}
+Retries the event type either from the media player event in case of v1 media schemas or the event name in case of v2 media schemas.
```

Comment:
```suggestion
Retrieves the event type either from the media player event in case of v1 media schemas or the event name in case of v2 media schemas.
```